src/devices/nm-device-bridge.c

```diff
@@ -168,26 +168,29 @@ complete_connection (NMDevice *device,
 typedef struct {
 	const char *name;
 	const char *sysname;
+	uint nm_min;
+	uint nm_max;
+	uint nm_default;
 	gboolean default_if_zero;
 	gboolean user_hz_compensate;
 } Option;
 
 static const Option master_options[] = {
-	{ NM_SETTING_BRIDGE_STP, "stp_state", FALSE, FALSE },
-	{ NM_SETTING_BRIDGE_PRIORITY, "priority", TRUE, FALSE },
-	{ NM_SETTING_BRIDGE_FORWARD_DELAY, "forward_delay", TRUE, TRUE },
-	{ NM_SETTING_BRIDGE_HELLO_TIME, "hello_time", TRUE, TRUE },
-	{ NM_SETTING_BRIDGE_MAX_AGE, "max_age", TRUE, TRUE },
-	{ NM_SETTING_BRIDGE_AGEING_TIME, "ageing_time", TRUE, TRUE },
-	{ NM_SETTING_BRIDGE_GROUP_FORWARD_MASK, "group_fwd_mask", TRUE, FALSE },
-	{ NM_SETTING_BRIDGE_MULTICAST_SNOOPING, "multicast_snooping", FALSE, FALSE },
+	{ NM_SETTING_BRIDGE_STP, "stp_state", 0, 1, 1, FALSE, FALSE },
+	{ NM_SETTING_BRIDGE_PRIORITY, "priority", 0, G_MAXUINT16, 0x8000, TRUE, FALSE },
+	{ NM_SETTING_BRIDGE_FORWARD_DELAY, "forward_delay", 0, NM_BR_MAX_FORWARD_DELAY, 15, TRUE, TRUE },
+	{ NM_SETTING_BRIDGE_HELLO_TIME, "hello_time", 0, NM_BR_MAX_HELLO_TIME, 2, TRUE, TRUE },
+	{ NM_SETTING_BRIDGE_MAX_AGE, "max_age", 0, NM_BR_MAX_MAX_AGE, 20, TRUE, TRUE },
+	{ NM_SETTING_BRIDGE_AGEING_TIME, "ageing_time", NM_BR_MIN_AGEING_TIME, NM_BR_MAX_AGEING_TIME, 300, TRUE, TRUE },
+	{ NM_SETTING_BRIDGE_GROUP_FORWARD_MASK, "group_fwd_mask", 0, 0xFFFF, 0, TRUE, FALSE },
+	{ NM_SETTING_BRIDGE_MULTICAST_SNOOPING, "multicast_snooping", 0, 1, 1, FALSE, FALSE },
```

Comment:
can you vertically align the properties, with spaces? This is hard hard to read.

Also, it's not clear to me what `in-memory mapped` here means.

Reply:
Ok, bridge options are now aligned.
Removed the "in-memory mapped" term.